monkey/infection_monkey/network/info.py

```diff
@@ -25,6 +25,10 @@ class NetworkAddress:
     domain: str
 
 
+def get_network_interfaces() -> List[IPv4Interface]:
+    return get_local_network_interfaces()
```

Comment:
Any reason not to rename this?

Reply:
Whoops, I didn't commit the file. I'll push it up